scripts/validate-model.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+PUBLISHER="${PUBLISHER:-../org.hl7.fhir.publisher.jar}"
+
+# For some reason, this generates a bunch of files owned by root which then cause problems later on
+# during the validation. 
+#fhirutil add ./site_root/input/resources
```

Comment:
Are you saying, if you run the publisher standalone, it generates a bunch of files owned by root?

Reply:
Well, the problem isn't the "fhirutil add", but the subsequent java call. It may be some weirdness related to WSL.